server/db/Office.js

```diff
@@ -0,0 +1,22 @@
+const { Model } = require('sequelize');
+const reOffice = /[a-z][0-9]{1,2}$/i;
```

Comment:
Well you are missing start of string `^` marker, could use `\d` instead of numeric group and this case insensitive modifier is quite nice hack but for me it is too easy to miss. :fox_face: